### PR TITLE
Error modal when uploading assets with already existing name

### PR DESCRIPTION
**Pull request by PierreGuilmin**
See https://github.com/Requarks/wiki/pull/3165, this PR cherry-picks the commits from the https://github.com/Requarks/wiki/pull/3165 PR on top of the last release deployed at Alice&Bob (`2.5.170`), to our custom `ab-2.5.170` branch.

:warning: We need to add a file at `server/locales/en.yml` with the following content for this PR to work:
```yml
editor:
  assets.conflictingFileNames: 'Conflicting file names, please rename the uploaded assets.'
```

### CODE REVIEW

File: client/components/editor/editor-modal-media.vue [hunk 1]
```diff
@@ -379,6 +379,14 @@ export default {
           icon: 'warning'
         })
       }
+      const conflictingFiles = _.intersectionWith(files, this.assets, (f, a) => f.filename === a.filename)
+      if (conflictingFiles.length > 0) {
+        return this.$store.commit('showNotification', {
+            message: this.$t('editor:assets.conflictingFileNames'),
```
Comment (nbourdau) [inline client/components/editor/editor-modal-media.vue:385]: I am not expert of the code base, neither in javascripts... But I have the feeling that "editor:assets.conflictingFileNames' is not defined. I bet this is a translation string you have created but you forgot to include in your commit.

Also, what about reporting which files already exist?

**Review by nbourdau — CHANGES_REQUESTED** [review]

File: client/components/editor/editor-modal-media.vue (hunk 1 above)
Comment (PierreGuilmin) [reply client/components/editor/editor-modal-media.vue:385]: Yes it is a translation string which does not live on this repository (that's why the name is not defined). The solution while waiting for the string to be added to the translation database is to add a locale `server/locales/en.yml`  file holding the English translation on our server, as explained in the PR description. I can't (and I think I shouldn't) commit this file as it denotes a specific server configuration, and matches one of the `.gitignore` regexp.

I wish we could say which files already exist in the error modal but the way translation has been coded doesn't allow passing variables to the message...

Comment (nbourdau) [reply client/components/editor/editor-modal-media.vue:385]: Sorry I overlook the PR message... But this information must lie in the commit message.

Ok for the inflexibility and the lack of fallback of the translation... It is really sad they don't use gettext like system...
Could you just create an additional commit that adds the local translation but that will not be pushed in the PR the official repo of Wiki.js?

Comment (PierreGuilmin) [reply client/components/editor/editor-modal-media.vue:385]: I fixed the commit message and added a commit with the local translation file 👍🏼

**Comment by PierreGuilmin** [comment]
Replaced by #2.

**Pull request closed without merging**